glossary/es/index.md

```diff
@@ -1,98 +1,6 @@
 ---
-redirect_from: /es/glossary.html
-section: glossary
+redirect_from: /en/glossary.html
+layout: glossary
```

Comment:
I actually think this page is created automatically and doesn't need to be touched. @pwalsh - what is your advice here?